src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
```

Comment:
jeśli dajesz typehint na wszystkie argumenty i w żaden sposób nie opisujesz ich w phpdocu – to po co phpdoc? nie wnosi nic ponad to co masz w kodzie. czy chodzi o wsparcie jakiś IDE, które jeszcze nie kumają czaczy z php7?